modules/ROOT/pages/munit-maven-plugin.adoc

```diff
@@ -704,17 +676,24 @@ MUnit allows you to specify the runtime version in which your applications being
     <artifactId>munit-maven-plugin</artifactId>
     <configuration>
       ...
-      <runtimeVersion>1.2.3</runtimeVersion>
+      <runtimeVersion>4.2.2</runtimeVersion>
       ...
     </configuration>
   </plugin>
 </plugins>
 ----
 
-== To Specify The Runtime Product
+== Specifying The JVM
 
-MUnit allows you to specify the type of runtime in which your applications being tested will run. +
-The two possible values are MULE for community edition, and MULE_EE for Enterprise Edition.
+MUnit allows you to specify the JVM (or the java executable) in which your applications being tested will run. You must populate the `munit.jvm` parameter with the path to the executable:
```

Comment:
MUnit allows you to specify > You can specify
in which your applications being tested will run > in which your tested applications run. 
You must populate > Populate